src/recipes/barraquito.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Barraquito",
+    "description": "A multi-layered coffee drink.",
+    "github": "RichardVerbraak",
+    "ingredients": [
+        "Coffee",
+        "Licor 43",
+        "Frothed Milk",
+        "Condensed Milk"
+    ],
+    "directions": [
+        "Step 1. Add the condensed milk to the coffee, followed by the licor.",
+        "Step 2. Add the espresso and optional lemon peel.",
+        "Step 3. Add the frothed milk and sprinkle some cinammon for some extra flavor."
+    ],
+    "image": "barraquito.jpg",
+    "source": "<url if available>",
+    "keywords": [
+        "Coffee",
+        "Spanish",
+        "Canary Isles"
```

Comment:
Can you make these lowercase? Thanks!

Reply:
Also done! I'm not sure how to update a PR but I added an updated file just now.